changelogs/unreleased/bump-libcurl-to-8.3.0.md

```diff
@@ -0,0 +1,3 @@
+## feature/build
+
+* Updated libcurl to version 8.3.0.
```

Comment:
```suggestion
* Updated libcurl to version 8.3.0 (gh-9086).
```

Reply:
This bug is specified in the second changelog. Bump was done without an issue.